### PR TITLE
4.0

### PR DESCRIPTION
**Pull request by mahabaleshwm**
Repair LVM information

### CODE REVIEW

**Comment by ssahani** [comment]
@mahabaleshwm Thanks for the patch. Please drop the first commit.and do a force push

**Comment by mikeroySoft** [comment]
The commits are also against the wrong branch, and the wrong docs entirely.

Recall that docs are in photon-hugo branch.

The /docs/ folder in the root of the main repo branch (which is still 'master', tho it should be migrated to 'main', but that's another issue), need to be deleted. I called this out [here]( https://github.com/vmware/photon/issues/1138 ).

The reason I haven't deleted that folder yet is that I'm unsure if it still gets used somehow.

Basically we have like 4 doc sites at this point, and we should be using only 1 (maybe 2...)
- Wiki <- only relevant thing here are the download links, everything else is out of date
- Old docs (https://vmware.github.io/photon/assets/files/html/3.0/) 
- /docs/ in :master
- photon-hugo docs **<- the one we should be using**

**Comment by mikeroySoft** [comment]
@mahabaleshwm, the path forward (for this and all doc changes) would be to make the changes to the markdown living in the photon-hugo branch (and no other branch).

Submit a PR against photon-hugo with just the doc changes and we'll get it merged in.

Changes to build.py should be done independently of doc changes, so that PR should get submitted via Gerrit (which I think it was?)

@ssahani Can you cherry pick https://github.com/vmware/photon/pull/1193/commits/e25020c1763c762221d99cb38c32b485e6731a3d ?

I'm

**Comment by ssahani** [comment]
> @mahabaleshwm, the path forward (for this and all doc changes) would be to make the changes to the markdown living in the photon-hugo branch (and no other branch).
> 
> Submit a PR against photon-hugo with just the doc changes and we'll get it merged in.
> 
> Changes to build.py should be done independently of doc changes, so that PR should get submitted via Gerrit (which I think it was?)
> 
> @ssahani Can you cherry pick [e25020c](https://github.com/vmware/photon/commit/e25020c1763c762221d99cb38c32b485e6731a3d) ?
> 
> I'm

The patch does not apply
